style.css

```diff
@@ -709,3 +709,282 @@ small {
+}
+.sidebar__item {
+    padding-top: 20px;
+}
+.sidebar__item:first-child {
+    padding-top: 0;
+}
+.sidebar__link,
+.tags__link {
+    text-decoration: none;
+    color: #F5F8FB;
+}
+.tags {
+    display: flex;
+    flex-wrap: wrap;
+    gap: 8px;
+}
+.tags__item {
+    border: 1px solid rgba(208, 215, 251, 0.17);
+    padding: 0 8px;
+}
+.post {
+    flex-shrink: 1;
+    flex-grow: 1;
+    flex-basis: auto;
+    font-size: 18px;
+    font-weight: 400;
+    color: #F8F7FF;
+}
+figure {
+    position: relative;
+    margin: 0 0 20px;
+}
+figcaption {
+    position: absolute;
+    z-index: 2;
+    bottom: 15px;
+    left: 25px;
+    font-size: 16px;
+    line-height: 24px;
+    font-weight: 500;
+    color: #FFF;
+}
+.content__title {
+    font: 36px/48px var(--font-secondary);
+    margin: 0 0 20px;
+}
+.post__btn {
+    margin-bottom: 32px;
+}
+.blog__info {
+    color: rgba(239, 245, 250, 0.82);
+}
+.post p {
+    margin: 0 0 16px;
+}
+.post ul {
+    margin: 0 0 16px;
+    padding-left: 16px;
+}
+.post li {
+    padding-top: 10px;
+}
+.post li:first-child {
+    padding-top: 0;
+}
+.post a {
+    text-decoration: none;
+    color: #78BBFA;
+    transition: opacity 0.2s linear;
+}
+.post a:hover {
+    opacity: 0.8;
+}
+.post .pointer-list {
+    padding: 0;
+    list-style: none;
+}
+.pointer-list li {
+    position: relative;
+    padding: 0 0 0 40px;
+    margin-top: 10px;
+}
+.pointer-list li:first-child {
+    margin-top: 0;
+}
+.pointer-list li::after {
+    content: '';
+    position: absolute;
+    top: 2px;
+    left: 0;
+    z-index: 2;
+    width: 22px;
+    height: 25px;
+    background-image: url(img/icons-logo.svg);
+}
+.blockquote {
+    background-color: #002754;
+    border-radius: 16px;
+    font-size: 24px;
+    line-height: 30px;
+    font-weight: 600;
+    text-transform: capitalize;
+    padding: 16px;
+    margin-bottom: 40px;
+}
+blockquote {
+    margin: 0 0 8px;
+}
+.cite {
+    font-size: 14px;
+    line-height: 20px;
+}
+.cite span{
+    position: relative;
+    padding-left: 16px;
+    color: rgba(239, 245, 250, 0.82);
+}
+.cite span::before {
+    content: "";
+    width: 16px;
+    height: 16px;
+    margin-right: 8px; 
+    background-image: url('img/stars.svg');
+    background-size: contain;
+    background-repeat: no-repeat;
+    position: absolute;
+    top: 2px;
+    left: 0;
+    z-index: 2;
+}
+.post__img {
+    display: flex;
+    gap: 24px;
+}
+.post__title {
+    font: 28px/36px var(--font-secondary);
+    margin: 0 0 20px;
+}
+.article-footer {
+    display: flex;
+    justify-content: space-between;
+}
+.article-footer__col {
+    display: flex;
+    align-items: center;
+}
+.article-footer__title {
+    font-size: 16px;
+    line-height: 24px;
+}
+.article-footer .tags {
+    margin: 0;
+    padding-left: 20px;
+}
+.article-footer .tags__item {
+    padding: 0 8px;
+}
+
+.article-footer .social-networks {
+    margin: 0;
+    padding-left: 20px;
+}
+.article-footer .social-networks li {
```

Comment:
This is so big selector. Lets try to reduce it

Reply:
Done